src/Raven.Server/ServerWide/Commands/Indexes/PutIndexCommand.cs

```diff
@@ -40,7 +41,14 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
                 {
                     throw new InvalidOperationException($"Can not add index: {Definition.Name} because an index with the same name but different casing already exist");
                 }
+
                 record.AddIndex(Definition, Source, CreatedAt, etag);
+
+                if (Definition.Staggered)
+                {
+                    if (InitializeStaggeredDeployment(record) == false) 
+                        return null;
```

Comment:
Can `InitializeStaggeredDeployment` method be a `void` method? We return `null` anyway here

Reply:
Yes.